CRM/Contact/BAO/Query.php

```diff
@@ -2253,6 +2255,7 @@ public function restWhere(&$values) {
       );
     }
     elseif ($name === 'addressee') {
+      CRM_Core_Error::deprecatedFunctionWarning('pass in addressee_id or ddressee_display');
```

Comment:
typo in "ddressee_display"

Reply:
yeah - but probably no-one will ever see that notice again until they go to remove the code....

Reply:
@colemanw ok - fixed